ui/src/components/stats/StateGlobalChart.vue

```diff
@@ -51,6 +59,30 @@
             },
             hasData() {
                 return this.count > 0;
+            },
+            daysCount() {
+                if (this.startDate && this.endDate) {
+                    return this.$moment(this.endDate).diff(this.$moment(this.startDate), "days") + 1;
+                }
+                return 31;
+            },
+            header() {
```

Comment:
This is not sweet for most of the case, just add the number and the type (https://github.com/kestra-io/kestra/issues/1240) 

- 123 execution
- 12 task runs

and remove the time interval